source/scpd/types/Stellar_SCP.d

```diff
@@ -161,7 +161,7 @@ struct SCPEnvelope {
 
 static assert(SCPEnvelope.sizeof == 168);
 
-struct SCPQuorumSet {
+extern(C++, class) struct SCPQuorumSet {
```

Comment:
Why?

Reply:
https://github.com/stellar/stellar-core/blob/dd7f38dba232e20ae75a95641857c0b7ed0252e1/src/xdr/Stellar-SCP.x#L80

This is not necessary because it has the same struct c++ side and d.
I'll put it back.